app/templates/frameworks/updates.html

```diff
@@ -108,9 +110,12 @@
       %}
         {% include "toolkit/forms/textbox.html" %}
       {% endwith %}
+      <p>
+        The deadline for clarification questions is x
```

Comment:
I think Chris has made this change in a separate pull request. The new content is in point 4. of these updates.

https://docs.google.com/document/d/1w6C90rIx8M3OIq5Ue7z0ef1ebb1qm0aueP8lBzH70l4/edit

https://preview.development.digitalmarketplace.service.gov.uk/suppliers/frameworks/g-cloud-7/updates

Under ‘Ask a G-Cloud 7 clarification question’, before the maximum word count, add:

‘The deadline for clarification questions is 5pm BST, 22 September 2015. All responses will be published by 5pm BST, 29 September 2015.’


On 27 Aug 2015, at 10:23, pcraig3 <notifications@github.com> wrote:

> In app/templates/frameworks/updates.html:
> 
> > @@ -108,9 +110,12 @@
> >        %}
> >          {% include "toolkit/forms/textbox.html" %}
> >        {% endwith %}
> > +      <p>
> > +        The deadline for clarification questions is x
> (This should probably be figured out soon.)
> 
> —
> Reply to this email directly or view it on GitHub.
>